packages/components/tests/dummy/app/templates/components/tabs.hbs

```diff
@@ -117,6 +129,24 @@
     '
   />
   {{! prettier-ignore-end }}
+
+  <h4 class="dummy-h4">With optional Count and Icon</h4>
+  {{! prettier-ignore-start }}
+  <CodeBlock
+    @language="markup"
+    @code='
+      <Hds::Tabs as |T|>
+        <T.Tab @count="5">One</T.Tab>
+        <T.Tab @icon="download">Two</T.Tab>
+        <T.Tab>Three</T.Tab>
+
+        <T.Panel>Content one <a href="#">Link to test tabbing</a></T.Panel>
+        <T.Panel>Content 2</T.Panel>
+        <T.Panel>Content 3!</T.Panel>
+      </Hds::Tabs>
+    '
+  />
+  {{! prettier-ignore-end }}
```

Comment:
[non-blocker] Not necessarily in this ticket, but can you add after these code examples also a render of the same code as component? see for example how it's done for these components:
- https://design-system-components-hashicorp.vercel.app/components/button#how-to-use
- https://design-system-components-hashicorp.vercel.app/components/form/radio#how-to-use
- https://design-system-components-hashicorp.vercel.app/components/alert#how-to-use
Thanks